src/utils/traffic-mesh.js

```diff
@@ -64,13 +77,98 @@ const startForwardProxy = async ({ port, frameworkPort, functionsPort, publishDi
   }
 }
 
+const forwardMessagesToLog = ({ log, subprocess }) => {
+  const { promise: firstBundleReady, reject: firstBundleReject, resolve: firstBundleResolve } = createDeferred()
+
+  let currentId = null
+  let spinner = null
+
+  const reset = () => {
+    currentId = null
+    spinner = null
+  }
+
+  const forwarder = rl.createInterface({
+    input: subprocess.stderr,
+  })
+
+  forwarder
+    .on('line', (line) => {
+      let data
+      try {
+        data = JSON.parse(line.trim())
+      } catch (error) {
+        log(`${NETLIFYDEVERR} Cannot parse log line as JSON: ${line.trim()}${EOL}${EOL}${error}`)
+        return
+      }
+
+      const { error, id, type } = data
+      switch (type) {
+        case 'bundle:start':
+          currentId = id
+          if (!spinner) {
+            spinner = startSpinner({ text: 'Bundling edge handlers...' })
+          }
+          break
+
+        case 'bundle:success':
+          if (currentId !== id) {
+            return
+          }
+
+          firstBundleResolve()
+          stopSpinner({ spinner, error: false, text: 'Done.' })
+          reset()
+          break
+
+        case 'bundle:fail':
+          if (currentId !== id) {
+            return
+          }
+
+          stopSpinner({
+            spinner,
+            error: true,
+            text: (error && error.msg) || 'Failed bundling Edge Handlers',
+          })
+          log(`${NETLIFYDEVLOG} Change any project file to trigger a re-bundle`)
```

Comment:
```suggestion
          log(`${NETLIFYDEVLOG} Change any project file to trigger an Edge Handlers re-bundle`)
```

Also a change to any project file would trigger a re-bundle? Or just changes to files under `edge-handlers`